tests_integration/spcs/test_image_repository.py

```diff
@@ -36,7 +38,6 @@ def _list_images(runner):
     )
 
 
-@pytest.mark.integration
```

Comment:
Why you removed mark? Now it will be executed as unit test

Reply:
This mark is actually a typo that should have been removed previously. If you expand the rest of this integration test, you'll see that _list_images and _list_tags are run as part of the `test_list_images_tags` test which is marked with pytest.mark.integration